plugins/zammad-mcp/skills/zammad-mcp-issue-digest/scripts/collect_issue_digest.py

```
@@ -0,0 +1,987 @@
+
+def summarize_issue(
+    issue,
+    comments,
+    requested_labels,
+    since,
+    until,
+    body_chars,
+    comment_chars,
+    issue_reaction_events=None,
+    comment_reactions_by_id=None,
+    all_labels=False,
```

Ruff (0.15.12)

[warning] Boolean default positional argument in function definition

(FBT002)